etc/bin/run.sh

```diff
@@ -30,7 +30,10 @@ KERNEL_ASSEMBLY=`(cd ${PROG_HOME}/lib; ls -1 toree-kernel-assembly-*.jar;)`
 
 # disable randomized hash for string in Python 3.3+
 export PYTHONHASHSEED=0
+# The SPARK_OPTS values during installation are stored in TOREE_SPARK_OPTS. This allows values to be specified during
+# install, but also during runtime.
+export SPARK_OPTS="${__TOREE_SPARK_OPTS__} ${SPARK_OPTS}"
```

Comment:
No, that would be needed for setting PATH, but not command line args. I am assuming these will expand into actual command line args.